.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -78,10 +78,13 @@ This is a checklist for PR authors & reviewers. Please make sure to complete all
 - [ ] If a new CSS style is added I verified that:
     - [ ] A similar style doesn’t already exist
     - [ ] The style can’t be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
+- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
+- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
 
 
+<details>
```

Comment:
Ah since the contributor and reviewer checklist are similar (we're still checking how to improve that) is better to keep the reviewer checklist collapsed because atm the reviewer can only copy and paste it in a comment to check it off (more context [here](https://expensify.slack.com/archives/C01SKUP7QR0/p1645014137518659?thread_ts=1643767911.674959&cid=C01SKUP7QR0)).

<img width="315" alt="Screen Shot 2022-03-22 at 12 30 39" src="https://user-images.githubusercontent.com/6829422/159551049-506a082e-bd50-44a2-bed1-e6174c669c9a.png">

Reply:
Nice, I didn't know about this collapsible fuctionality